justfile

```diff
@@ -0,0 +1,180 @@
+
+# Make sure that some env variables are set for all jobs.
+#FIXME: Consider using .env for this, but only if used also on CI
```

Comment:
I thought we decided to use it. Could be a problem with the CI?

Reply:
just not in this PR, the comment is older then the decision

Reply:
updated the fixme